tests/framework/ROM/SKLearn/tests

```diff
@@ -147,7 +147,7 @@
     input = 'gp.xml'
     csv = 'data/outGP.csv'
     output = 'data/outGP.xml'
-    rel_err = 1.0e-6
+    rel_err = 5.0e-6
```

Comment:
we know why but can you comment why we changed the tolerance?

Reply:
updated

Reply:
:thumbsup: